src/humanloop/requests/version_specification.py

```diff
@@ -0,0 +1,37 @@
+# This file was auto-generated by Fern from our API Definition.
+
+import typing_extensions
+import typing_extensions
```

Comment:
Check where this is used

Reply:
no longer in SDK now with updated openapi spec